MIGRATING.md

```diff
@@ -0,0 +1,57 @@
+# Migrating
+
+## From version 2.x to 3.x
+
+### Full `@okta/okta-auth-js` API is available
+
+`@okta/okta-angular` version 2.x and earlier provided a wrapper around [@okta/okta-auth-js](https://github.com/okta/okta-auth-js) but many methods were hidden. Version 3.x extends `okta-auth-js` so the full [api](https://github.com/okta/okta-auth-js/blob/master/README.md#api-reference) and all [options](https://github.com/okta/okta-auth-js/blob/master/README.md#configuration-options) are now supported by this SDK. To provide a better experience, several methods which existed on the wrapper have been removed or replaced.
+
+### "Active" token renew
+
+Previously, tokens would only be renewed when they were read from storge. This typically occurred when a user was navigating to a protected route. Now, tokens will be renewed in the background before they expire. If token renew fails, the [authState](https://github.com/okta/okta-auth-js/blob/master/README.md#authstatemanager) will be updated and `authState.isAuthenticated` will be recalculated. If the user is currently on a protected route, they will need to re-authenticate. Set the `onAuthRequired` option to customize behavior when authentication is required. You can set [tokenManager.autoRenew](https://github.com/okta/okta-auth-js/blob/master/README.md#autorenew) to `false` to disable active token renew logic.
```

Comment:
Looks like there are lots of references of `authState` in this doc, how about move the link to the top/bottom as a link reference.
```
[authState]: https://github.com/okta/okta-auth-js/blob/master/README.md#authstatemanager
```
then use `[authState][]` in the md